pincer/objects/guild/guild.py

```diff
@@ -486,6 +487,207 @@ async def kick(self, member_id: int):
         """
         await self._http.delete(f"guilds/{self.id}/members/{member_id}")
 
+    async def get_roles(self):
+        """|coro|
+        Fetches all the roles in the guild.
+
+        Returns
+        -------
+        List[:class:`~pincer.objects.guild.role.Role`]
+            A list of Role objects.
+        """
+        data = await self._http.get(f"guilds/{self.id}/roles")
+        return [Role.from_dict(construct_client_dict(self._client, i)) for i in data]
+
+    @overload
+    async def create_role(
+        self,
+        *,
+        name: Optional[str] = "new role",
+        permissions: Optional[str] = None,
+        color: Optional[int] = 0,
+        hoist: Optional[bool] = False,
+        icon: Optional[str] = None,
+        unicode_emoji: Optional[str] = None,
+        mentionable: Optional[bool] = False
+    ) -> Role:
+        """|coro|
+        Creates a new role for the guild.
+        Requires the `MANAGE_ROLES` permission.
+
+        Parameters
+        ----------
+        name : Optional[:class:`str`]
+            name of the role |default| data:`"new role"`
```

Comment:
```suggestion
        name : Optional[:class:`str`]
            name of the role |default| :data:`"new role"`
```
This change should made for the rest of the datas ofc